docs/rules/prefer-array-index-of.md

```diff
@@ -0,0 +1,31 @@
+# Prefer `Array#indexOf()` over `Array#findIndex()` when searching the index of an item
```

Comment:
This should be the same as the readme description.

Reply:
This is not done.

Reply:
I don't understand. I would say [this](https://github.com/Fabrice-TIERCELIN/eslint-plugin-unicorn/blob/prefer-indexof-over-findindex/docs/rules/prefer-array-index-of.md#prefer-arrayindexof-over-arrayfindindex-when-searching-the-index-of-an-item) is the _readme description_. Or you want to say this line should be the same as the description below, as this is a title. Or you wanna say it should be the same as the [message id](https://github.com/Fabrice-TIERCELIN/eslint-plugin-unicorn/blob/prefer-indexof-over-findindex/rules/prefer-array-index-of.js#L10).

I thought you wanted to say that the link to mozilla were not formatted the same way as below.

Reply:
In readme.md it's

> - [prefer-array-index-of](docs/rules/prefer-array-index-of.md) - Prefer `.includes()` over `.indexOf()` when checking for existence or non-existence. *(fixable)*